Basic-Car-Maintenance-UITests/Extensions/XCUIApplication-dismissPopup.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  XCUIApplication-dismissPopup.swift
+//  Basic-Car-Maintenance
+//
+//  https://github.com/mikaelacaron/Basic-Car-Maintenance
+//  See LICENSE for license information.
+//
+
+import XCTest
+
+extension XCUIApplication {
+    func dismissPopup() {
+        otherElements["dismiss popup"].tap()
```

Comment:
change this function to `dismissTipKitPopup`

but before that, question, does this only work for TipKit, or what other "popups" would this function be used for? if it's specific to this test, I would actually move it so it's in the `BasicCarMaintenanceUITests` class